monai/eval_savers/csv_saver.py

```diff
@@ -0,0 +1,81 @@
+# Copyright 2020 MONAI Consortium
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#     http://www.apache.org/licenses/LICENSE-2.0
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import os
+import csv
+import numpy as np
+import torch
+
+
+class CSVSaver:
+    """
+    save the data in a dictionary format cache, and write to a CSV file finally.
+    Typically, the data can be classification predictions.
```

Comment:
need some documentation about how to use `save` `save_batch` `finalize`

Reply:
updated in the latest patch.
Thanks.